build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/test/rest/RestTestBasePlugin.java

```diff
@@ -323,6 +323,8 @@ private Optional<String> findModulePath(Project project, String pluginName) {
 
     private Configuration createPluginConfiguration(Project project, String name, boolean useExploded, boolean isExtended) {
         return project.getConfigurations().create(name, c -> {
+            c.setCanBeResolved(true);
```

Comment:
Not having this caused weirdness with composite builds attempting to resolved exploded plugin bundles.